virtool/api/response.py

```diff
@@ -26,27 +37,72 @@ def json_response(
 
 
 class InsufficientRights(HTTPForbidden):
+    """
+    Reusable exception for returning an HTTP status `403` (Insufficient Rights) message
+    to the client.
```

Comment:
* How is it used?
  > Raising this exception during request handling immediately returns a `403` (Insufficient Rights) response to the client.
* "Reusable" doesn't have much meaning here.
* Please use this wording for the other docstrings.